shell/utility/electron_content_utility_client.cc

```diff
@@ -115,6 +121,8 @@ void ElectronContentUtilityClient::RegisterMainThreadServices(
     (BUILDFLAG(ENABLE_PRINTING) && BUILDFLAG(IS_WIN))
   services.Add(RunPrintingService);
 #endif
+
+  services.Add(RunNodeService);
```

Comment:
Does this theoretically mean that multiple node services could be run in the same process as each other / as printing/etc. services? or are these one-per-process?

Reply:
These are one per process.